jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/ObjectRule.java

```diff
@@ -34,7 +34,7 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
-import com.sun.codemodel.JExpression;
+import com.sun.codemodel.*;
```

Comment:
Oops, that was autogenerated by IntelliJ since this workspace doesn't have our style rules. I'll change it back.